fetch.bs

```diff
@@ -4792,7 +4792,11 @@ steps:
  <a for="fetch params">process response</a> given <var>response</var>, with <var>fetchParams</var>'s
  <a for="fetch params">task destination</a>.
 
- <li><p>If <var>response</var>'s <a for=response>body</a> is null, then run
+ <li><p>Let <var>actualResponse</var> be <var>response</var>'s
+ <a for="filtered response">internal response</a> if <var>response</var> is a
+ <a>filtered response</a>; Otherwise <var>response</var>.
```

Comment:
1. Since this is after main fetch it's always going to be a filtered response or a network error. Hence we should have a step equivalent to step 15 of main fetch. (Might want to update the wording though as this does if/otherwise better, except that otherwise needs to be lowercase.)
2. We should use the same variable name throughout the specification. We use both actualResponse and internalResponse. _internalResponse_ seems clearer to me.

Reply:
Done